editor/src/core/shared/element-path.ts

```diff
@@ -8,7 +8,6 @@ import {
 import { arrayEquals, longestCommonArray, identity, fastForEach } from './utils'
 import { replaceAll } from './string-utils'
 import { last, dropLastN, drop, splitAt, flattenArray, dropLast } from './array-utils'
-import { extractOriginalUidFromIndexedUid } from './uid-utils'
```

Comment:
was this change needed to make it compile? or just an improvement related to your example test case?

Reply:
It was a change needed to avoid a circular reference.

Reply:
oh boy

Reply:
So this would be the circular reference that has bitten us a number of times in tests!